src/components/Label/Label.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react'
+import FormLabel from 'react-bootstrap/FormLabel'
+
+interface Props {
+  /** Text to display in label */
+  text: string
+}
+
+/**
+ * Labels are used to display textnpm
+ */
+class Label extends Component<Props, {}> {
```

Comment:
Would this component be better as a functional component?

Reply:
Initially I was going to use a functional component here but after looking at some of the other bootstrap components (i.e textInput, textField, radio, badge, ect) I noticed that we tend towards using React Components. I wasn't sure if it was an explicit decision or not so I choose to follow the paradigm in the other components on this initial cut. 

I'm happy to change this component to a functional one for now though, what are you thoughts on adding a ticket for refactoring the other bootstrap components to functional components?

Reply:
I actually refractored  most of the components last night 😀 https://github.com/HospitalRun/components/pull/85.